docs/mine-hnt/validators/monitoring.mdx

```diff
@@ -41,4 +41,25 @@ If you’re syncing, something similar to the following should appear:
 14        449
 ```
 
-The first number is the **election epoch** and the second number is the **block height** of your validator. 
+The first number is the **election epoch** and the second number is the **block height** of your validator. The **height** will be within 100 of the number displayed on [https://explorer.helium.com/](https://explorer.helium.com/) if your validator is caught up. This may take a few days if your validator is new.
```

Comment:
Consider rephrasing this.  "To be considered 'online' in Explorer(link) and be eligible for election into a consensus group, the ** block height* needs to be with 100 blocks of the block height of the chain.  You can view the current block height of the chain through the Explorer(link).  Block heights may sync slowly, and may take a few days if your validator is just coming online."

Reply:
pushing a rephrase to this. It's yours with a few small changes.